### DISCUSSION TITLE
UI有可以优化的地方吗？

### DISCUSSION THREAD

**Discussion by Bronya0** [Bronya0/Kafka-King#4] [Q&A]
希望大家提一点建议

**Comment by ieayoio** [comment]
我有两点建议：
1.希望增加断开连接的功能
2.未连接时，左侧的节点、主题、生产者消费者等与节点有关的部分建议不要显示。

目前左侧边栏很多按钮实际上是对某个节点操作的，但是和某个节点的关联感不是很强
